source/jucePlugin/VirusController.cpp

```diff
@@ -945,7 +945,7 @@ namespace Virus
 
 	juce::String numToKeytrigger(float v, Parameter::Description)
 	{
-		return juce::roundToInt(v) == 0 ? "Off" : "Keytrigger Phase " + juce::String(juce::roundToInt(v));
+		return juce::roundToInt(v) == 0 ? "Off" : juce::String(juce::roundToInt(v));
```

Comment:
What's the fix here? Remember that exposed parameters are also used for automation lanes.

Reply:
hmm.. I didnt realize that automation issue i just created here..  i dont know the correct answer for this..     the problem with the current  implemtation is the value does not show up in the box next to the slider

Reply:
Start playing with git rebase -i and the oldest commit you'd like to start "rewriting history" for.

Just remember you'll need to force push to this branch afterward as they'll diverge.

Reply:
noted..  will def be spending some time this week learning to properly work git...  i know there are a lot of gotchas if you don't think things through ahead of time.

Reply:
back to this specific edit though  what to you propose  the correct solution is?